modules/virt-about-vmis.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+//
+
+[id="virt-about-vmis_{context}"]
+= About virtual machine instances
+
+A virtual machine instance (VMI) is a representation of a running virtual machine (VM). When a VMI is owned by a VM or another object, it is managed through its owner in the web console or by using the `oc` command-line interface (CLI).
+
+A standalone VMI is created and started independently with a script, through automation, or other methods in the CLI. In your environment, you might have standalone VMIs that were developed and started outside of the OpenShift virtualization environment. You can continue to manage those standalone VMIs by using the CLI. Alternatively, you can use the web console for specific tasks associated with standalone VMIs:
+
+* List standalone VMIs and their details.
+
+* Edit labels and annotations for a standalone VMI.
```

Comment:
As you said "labels"  cannot be edited because of a defect, should remove "labels" here.

Reply:
I have posted questions regarding label editing in the Jira story. Please review and respond in the Jira story. Thank you!

Reply:
Thanks for responding and providing access to the CNV 2.4 lab environment. I have validated the content with the UI observed in the web console. Updated the documentation to show that labels can be edited from the Actions menu. Also updated the documentation for navigation which is now Workloads > Virtualization. Please review. If everything looks good, please approve and update the Jira story. Thank you!

Reply:
Looks good.